templates/solaredge.yaml

```diff
@@ -43,22 +43,16 @@ pred_bat:
   #  - number.givtcp_{geserial}_load_power
   soc_kw:
     - sensor.solaredge_b1_available_energy
-  #soc_percent:
-  #  - sensor.solaredge_b1_state_of_energy
```

Comment:
any reason this has been commented out?

Reply:
`available_energy` is a health metric (don't worry, it confused me at first too!), calculated by `maximum_energy * (state_of_health / 100)` - it's not always guaranteed to be available (at least on my battery) so `soc_max` should probably be set to `sensor.solaredge_b1_maximum_energy` (as it is) and available energy in KWh calculated backwards from `state_of_energy`

Reply:
So I have put back soc_percent for now, this seems correct?

Maybe you can confirm this configuration actually works out of the box?

Reply:
Here's the full block as I have it:
```yaml
  battery_rate_max:
    - sensor.solaredge_b1_max_charge_power
  charge_rate:
    - number.solaredge_i1_storage_charge_limit
  discharge_rate:
    - number.solaredge_i1_storage_discharge_limit
  battery_power:
    - sensor.solaredge_b1_dc_power
  pv_power:
    - sensor.solar_panel_production_w # This is a custom sensor - see Readme on how to create or comment out
  load_power:
    - sensor.solar_house_consumption_w # This is a custom sensor - see Readme on how to create or comment out
  soc_percent:
    - sensor.solaredge_b1_state_of_energy
  soc_max:
    - sensor.solaredge_b1_maximum_energy
  # reserve:
  #   - number.solaredge_i1_backup_reserve
  #inverter_mode:
  # - select.givtcp_{geserial}_mode
  #inverter_time:
  # - sensor.givtcp_{geserial}_invertor_time
  charge_start_time:
   - "00:00:00"
  charge_end_time:
   - "00:01:00"
  charge_limit:
   - number.solaredge_i1_storage_charge_limit
  scheduled_charge_enable:
   - off
  scheduled_discharge_enable:
   - off
  discharge_start_time:
   - "00:00:00"
  discharge_end_time:
   - "00:01:00"
```

so the only things that differ from your implementation as it stands are `battery_rate_max` (uses sensor instead of assumption), `battery_power`, and (`pv_power` & `load_power`) (using custom sensors) - hope that helps!

Reply:
I've updated as you mentioned